### ISSUE TITLE
account command

### ISSUE THREAD

**Issue by Maxime2** [Fantom-foundation/go-evm#19]
Do we need account command for evm similar one of eth to be able to create and manage new accounts, on testnet in particular?

**Comment by andrecronje** [comment]
Good idea, let's add it to the backlog. Not sure if we should continue with go-evm or use the go-ethereum with lachesis integrated that we already built. https://github.com/Fantom-foundation/go-ethereum/tree/lachesis
